### ISSUE TITLE
✅ ⛔ 🔗 🐞 Bug Report: TinyMCE top bar (Text editor widget) gets overlapped by the Elementor Top Bar when clicking Full Screen (Duplicate of #22873) - [ED-11865]

### ISSUE THREAD

**Issue by toineenzo** [elementor/elementor#23463]
### Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When the Top bar experiment is enabled and you drop in a text editor widget, when you click the fullscreen button from the text editor bar, the bar with Paragraph and other text styling option falls behind the Elementor Top bar.
![image](https://github.com/elementor/elementor/assets/4921397/ef94f670-1cdb-445a-8b48-96b32b0974db)

Please see the following video:
https://github.com/elementor/elementor/assets/4921397/11985276-c58b-4fa4-8735-11bccb1c5a8a

### Steps to reproduce

1. Enable Top bar experiment
2. Open editor and drop in a Text editor widget
3. Click the Fullscreen button in the Text / WYSIWYG / TinyMCE editor, the bar of the text editor gets overlapped by the Elementor Top bar
4. You can only Exit Fullscreen mode by temporarily adding the following CSS to the Top bar: "display: none;" 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
Operating System: Linux
Software: Apache/2.4.57 (Ubuntu) mod_fcgid/2.3.9 OpenSSL/1.1.1f
MySQL version: mariadb.org binary distribution v10.6.14
PHP Version: 8.1.20
PHP Memory Limit: 128M
PHP Max Input Vars: 1000
PHP Max Post Size: 256M
GD Installed: Yes
ZIP Installed: Yes
Write Permissions: All right
Elementor Library: Connected

== WordPress Environment ==
Version: 6.3
Site URL: https://oor-pigeon-laso.instawp.xyz
Home URL: https://oor-pigeon-laso.instawp.xyz
WP Multisite: No
Max Upload Size: 256 MB
Memory limit: 128M
Max Memory limit: 128M
Permalink Structure: /%postname%/
Language: en-US
Timezone: 0
Debug Mode: Inactive

== Theme ==
Name: Hello Elementor
Version: 2.8.1
Author: Elementor Team
Child Theme: No

== User ==
Role: administrator
WP Profile lang: en_US
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

== Active Plugins ==
Elementor
Version: 3.15.2
Author: Elementor.com

== Elementor Experiments ==
Optimized DOM Output: Active
Improved Asset Loading: Active
Improved CSS Loading: Active
Inline Font Icons: Active
Additional Custom Breakpoints: Active
admin_menu_rearrangement: Inactive by default
Flexbox Container: Active
Upgrade Swiper Library: Active
Grid Container: Active
Hello Theme Header & Footer: Active
Editor Top Bar: Active
Landing Pages: Active
Nested Elements: Active
Lazy Load Background Images: Inactive by default
Global Style Guide: Active

== Log ==
PHP: showing 1 of 1PHP: 2023-08-17 16:41:15 [notice X 2][../wp-content/plugins/elementor/core/upgrade/manager.php::63] version_compare(): Passing null to parameter #1 ($version1) of type string is deprecated [array (
'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==

````

</details>

**Comment by nicholaszein** [comment]
Hello @toineenzo!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #22873

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original. Refer to the original post for more information about this subject.

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hi @toineenzo!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.17! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
